api/build.gradle

```diff
@@ -35,6 +35,11 @@ task copyCommitRunScripts(type: Copy) {
     into "$rootDir/api/src/main/resources/static/commit-run-scripts"
 }
 
+task copyMonitoringTemplate(type: Copy) {
```

Comment:
I think that by default we can store the template in the `api/src/main/resources/templates/` folder and use `classpath:templates/...` url as default value for `monitoring.stats.export.xls.template`. In this case you do not need any additional file manipulations during build and deploy and you still can replace xls template with some external value setting `CP_API_MONITORING_XLS_TEMPLATE_PATH` env var